shared/react-native/ios/Keybase/NativeLogger.m

```diff
@@ -0,0 +1,42 @@
+#import "NativeLogger.h"
+#import <UIKit/UIKit.h>
+#import "AppDelegate.h"
+
+@import CocoaLumberjack;
+@implementation NativeLogger
+
+static const DDLogLevel ddLogLevel = DDLogLevelDebug;
+static const NSString* tagName = @"KBNativeLogger";
+
+RCT_EXPORT_MODULE(KBNativeLogger);
+
+RCT_REMAP_METHOD(log,
+                 tagPrefix:(NSString*)tagPrefix
+                 toLog:(NSString*)toLog)
+{
+  DDLogInfo(@"%@%@: %@", tagPrefix, tagName, toLog);
+}
+
+RCT_REMAP_METHOD(dump,
+                 tagPrefix:(NSString*)tagPrefix
+                 resolver:(RCTPromiseResolveBlock)resolve
+                 rejecter:(RCTPromiseRejectBlock)reject)
+{
+  NSMutableArray<NSString *> *lines = [[NSMutableArray alloc] init];
```

Comment:
Do we have ARC turned on? Otherwise, we might have to worry about manual memory management...

Reply:
yeah arc is on